src/models/data/data-set.ts

```diff
@@ -666,23 +757,29 @@ export const DataSet = types.model("DataSet", {
         });
       },
 
-      selectCases(caseIds: string[], select = true) {
-        if (select) self.attributeSelection.clear();
-        const ids: string[] = [];
-        caseIds.forEach(id => {
-          const pseudoCase = self.pseudoCaseMap[id];
-          if (pseudoCase) {
-            ids.push(...pseudoCase.childCaseIds);
-          } else {
-            ids.push(id);
-          }
-        });
-        ids.forEach(id => {
+      selectCases,
+
+      selectCells(cells: ICell[], select = true) {
+        if (select) {
+          self.attributeSelection.clear();
+          self.caseSelection.clear();
+        }
+
+        if (getAppConfig(self)?.getSetting("cellsSelectCases", "dataset")) {
+          selectCases(uniqueCaseIds(cells), select);
```

Comment:
If two cells in a case are selected and you unselect one of them, will this unselect the case? Is that desirable?